spec/spec_helper.rb

```diff
@@ -151,11 +152,10 @@ def main_app
   Valkyrie::Persistence::Memory::MetadataAdapter.new, :test_adapter
 )
 
-Valkyrie::MetadataAdapter
-  .find(:test_adapter)
-  .query_service
-  .custom_queries
-  .register_query_handler(Hyrax::CustomQueries::FindAccessControl)
+query_service = Valkyrie::MetadataAdapter.find(:test_adapter).query_service
+custom_queries = [Hyrax::CustomQueries::Navigators::FindFiles,
+                  Hyrax::CustomQueries::FindAccessControl]
+custom_queries.each { |query_handler| query_service.custom_queries.register_query_handler(query_handler) }
```

Comment:
There's a related change in https://github.com/samvera/hyrax/pull/4082/files.

It might be worth cherry picking that version of this work?

Reply:
DONE - integrated the change in #4082 here.